docs/getting-started/deploy-singlenode-cluster.md

```diff
@@ -0,0 +1,103 @@
+    
+    During deployment, a cluster network called the [*management network*](../networking/index#built-in-cluster-network) is created for intra-cluster communications. Harvester allows you to create [custom cluster networks](../networking/index#custom-cluster-network) that can be dedicated to VM traffic and that allow VMs to be accessed from both internal and external networks.
+
+- [**Network configuration**](../networking/index#cluster-network): Definition of how cluster nodes connect to a specific cluster network. 
+    
+    Each network configuration corresponds to a set of nodes with uniform network specifications. Only nodes that are covered by the network configuration can access the associated cluster network. This arrangement offers you flexibility when configuring a heterogeneous cluster, particularly when the network interface names are different for each node.
+
+- [**VM network**](../networking/index#vm-network): Virtual network that VMs use to communicate with other VMs and external networks.
+    
+    Each VM network is linked to a specific cluster network, which is used for transmission of VM traffic. You can create either a [VLAN network](../networking/harvester-network/#vlan-network) or an [untagged network](../networking/harvester-network/#untagged-network) based on your requirements, such as traffic isolation, network segmentation, ease of management, or alignment with the external network environment. 
+
+Before you create VMs, create the necessary networks. If more than one network interface is attached to each cluster node, consider [creating custom cluster networks](../networking/index#how-to-create-a-new-cluster-network) and network configurations for better traffic isolation. Otherwise, you can only use the management network for transmission of VM traffic. Next, [create a VM network](../networking/index#how-to-create-a-new-cluster-network) that is linked to either the management network or any of the custom cluster networks that you created.
+
+## 8. Import VM images. 
+
+On the Harvester UI, you can import ISO, qcow2, and raw [images](../upload-image/) by uploading an image from the local file system, or by specifying the URL of an image that can be accessed from the cluster. 
+
+## 9. Import SSH keys. (Recommended)
+
+You can store SSH public keys in Harvester. When a VM is launched, a stored key can be [injected](../vm/access-to-the-vm/#ssh-access) into the VM to allow secure access via SSH. Validated keys are displayed on the **SSH Keys** screen on the Harvester UI. 
+
+## 10. Create VMs. 
+
+You can create [Linux VMs](../vm/index) using one of the following methods: 
+
+- Harvester UI: On the **Virtual Machines** screen, click **Create** and configure the settings on each tab. 
+- Kubernetes API: Create a `VirtualMachine` object. 
+- Harvester Terraform Provider: Define a `harvester_virtualmachine` resource block. 
```

Comment:
Can provide link to `terraform/terraform-provider`
e.g. https://6695198c1a3755207d5942de--harvester-preview.netlify.app/v1.3/terraform/terraform-provider/